nix/tests/lanzaboote.nix

```diff
@@ -147,7 +162,32 @@ let
         };
         boot.lanzaboote = {
           enable = true;
-          enrollKeys = lib.mkDefault true;
+          # Under aarch64, various things goes wrong...
```

Comment:
nit: You could elaborate here a bit, so people know why this wrinkle exists and when it may go away.